### DISCUSSION TITLE
How to send a request to the Admin REST API with a JWT token?

### DISCUSSION THREAD

**Discussion by Christopher-06** [apache/pulsar#19104] [Q&A]
Hello together,

I am trying to configure my Pulsar cluster and wanted to create a Python script for it. But since I have JWT token enabled for security reasons, it needs to be sent along to the REST API, but I don't know where it needs to be set. 

Does anyone here know more specific?
Cheers!

**Comment by yangl** [comment]
curl -X GET -H 'Authorization: Bearer JTW_TOKEN(NO token: prefix)' -H 'X-Pulsar-Auth-Method-Name: token' -H 'Content-Type: application/json; charset=UTF-8' -i 'http://ServiceURL:PORT/admin/v2/persistent/public/default/partitioned?includeSystemTopic=false'

**Comment by dhillonr** [comment]
```
import os
import requests

def read_token(path):
    if not os.path.exists(path):
        raise FileNotFoundError(f"File {path} does not exist")
    with open(path, 'r') as f:
        return f.read().strip()

path=<path_to_jwt_token_file>
BROKER = 'http://<ServiceURL>:<PORT>'
headers = {
    "Authorization": f"Bearer {read_token(path)}"
}   

response = requests.get(f'{BROKER}/admin/v2/brokers/version', headers=headers)
print("Check version:", response.status_code, response.text)
